### PR TITLE
contract comments

### PR DESCRIPTION
**Pull request by andrew-blox**

### CODE REVIEW

File: contracts/SSVNetwork.sol [hunk 1]
```diff
@@ -146,8 +146,8 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
                 ++dao.validatorCount;
                 _dao = dao;
             }
-
-            require(!_liquidatable(pod.balance, pod.validatorCount, operatorIds), "account liquidatable");
+            // IMPROPER ERROR SCHEMA ALSO WHY NOT IN  _validateValidatorParams FUNCTION
```
Comment (vadiminc) [inline contracts/SSVNetwork.sol:149]: @andrew-blox there are different types of validation.
- validation for the input func params
- business logic validation (for instance, is liquidatable or not)

thats not good idea to merge them into one place.

File: contracts/SSVNetwork.sol [hunk 2]
```diff
@@ -255,6 +255,7 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         emit ValidatorRemoved(validatorPK, clusterId);
     }
 
+    // WHY NO VALIDATION ON AMOUNT?  WHAT IF NEW POD IS TOO EXPENSIVE AND DOES IMMEDIATE LIQUIDATION OR EVEN SKIPS LIQUIDATION AND GOES TO NEGATIVE
```
Comment (vadiminc) [inline contracts/SSVNetwork.sol:258]: @andrew-blox  i think transfer is not completed logic, no sense to review it, it's just first impl version

File: contracts/SSVNetwork.sol [hunk 3]
```diff
@@ -279,6 +280,8 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
             }
             // Changing to a single event reducing by 15K gas
         }
+
+        // WHY THIS EMIT IS NOT LAST THING OR AT LEAST AFTER VALIDATION
```
Comment (vadiminc) [inline contracts/SSVNetwork.sol:284]: @andrew-blox in solidity it doesn't matter - if the validation will be failed, the func flow will be reverted

Comment (andrew-blox) [reply contracts/SSVNetwork.sol:284]: Yeah @lior-blox updated me as well on that, i personally still think its best to keep last just to follow same format as other functions.

**Pull request closed without merging**